app/src/main/java/com/google/android/apps/youtube/app/watchwhile/WatchWhileActivity.java

```diff
@@ -0,0 +1,7 @@
+package com.google.android.apps.youtube.app.watchwhile;
```

Comment:
This is the wrong package for dummy classes. Move this class to `https://github.com/revanced/revanced-integrations/tree/main/dummy/src/main/java/com/google/android/apps/youtube/app`.